Packages/com.unity.inputsystem/InputSystem/Monobehaviours/TrackedPoseDriverNIS.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.Controls;
+using UnityEngine.Experimental.XR.Interaction;
+
+namespace UnityEngine.XR.Experimental
+{
+
+    [DefaultExecutionOrder(-30000)]
+    [Serializable]
+    [AddComponentMenu("XR/Tracked Pose Driver (New Input System)")]
+    public class TrackedPoseDriverNIS : MonoBehaviour

```

Comment:
Please add XML docs.

Reply:
Can we name this just `TrackedPoseDriver`? If this conflicts with the name of the old name, IMO that's fine if we plan to obsolete that one.

BTW note that we have no migration mechanism for when classes are moved around. When the new input stuff moves out of experimental, existing components will likely break.